.github/ISSUE_TEMPLATE/new_dns_provider.yml

```diff
@@ -0,0 +1,44 @@
+name: New DNS provider support
+description: Request for the support of a new DNS provider.
+title: "[Bug]: "
```

Comment:
I wouldn't classify the request for a new provider a "bug". The title should reflect that.

Maybe `Support for new provider [provider name here]`?

Reply:
oops!